README.md

```diff
@@ -267,3 +268,23 @@ Warning:       No code coverage driver available
 
 nekufa@fasiga ~ % cat /proc/cpuinfo | grep i5
 model name  : Intel(R) Core(TM) i5-4670K CPU @ 3.40GHz
+```
+
+## Configuration Options
+
```

Comment:
Could you please include all the existing config options, including jwt and nkey?

Reply:
I'm not 100% sure how they work. Are they alternatives to CredentialsParser?

Reply:
CredentialsParser just returns a config array with nkey and/or jwt, but they can be defined directly

https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro/jwt

JWT: JSON Web Token ( default null )
NKey: Ed25519 based public key signature ( default null )

Reply:
I've added entries for jwts and nkeys to the latest.